### PR TITLE
feat: migrate the history tab components to fluent UI v9

### LINKED ISSUE #3443: Migrate Components in History

### PR DESCRIPTION
**Pull request by musale**
## Overview
Migrates components in the History tab to fluent UI v9.
Closes #3443

### CODE REVIEW

File: src/app/utils/dynamic-sort.spec.ts [hunk 1]
```diff
@@ -33,30 +36,7 @@ describe('Dynamic Sort', () => {
       { name: 'Ann', age: 14 }
     ];
 
-    const sortedArray = arrayToSort.sort(dynamicSort('name', SortOrder.DESC));
+    const sortedArray = arrayToSort.sort(dynamicSort<INameAge>('name', SortOrder.DESC));
```
Comment (thewahome) [inline src/app/utils/dynamic-sort.spec.ts:39]: I love this!

File: src/app/views/sidebar/history/HistoryV9.tsx [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+    dispatch(setSampleQuery(sampleQuery));
+    dispatch(setQueryResponse({
+      body: query.result,
+      headers: query.responseHeaders
+    }))
+    // TODO: change the message bar type to v9 types
+    dispatch(setQueryResponseStatus({
+      duration,
+      messageType:
+        status < 300 ? MessageBarType.success : MessageBarType.error,
```
Comment (Mnickii) [inline src/app/views/sidebar/history/HistoryV9.tsx:215]:
```suggestion
        status < 300 ? 'success' : 'error',
```

**Review by Mnickii — COMMENTED** [review]
Let's use the v9 types here

File: src/app/views/sidebar/history/HistoryV9.tsx (hunk 2 above)
Comment (Mnickii) [reply src/app/views/sidebar/history/HistoryV9.tsx:215]: Actually, this can be migrated as part of the Status migration, nvm!

**Review by Mnickii — APPROVED** [review]

**Pull request merged**